storybook/webpack.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
     loaders: [
       {
         test: /\.css$/,
-        loader: 'style',
+        loader: 'style?insertAt=top',
```

Comment:
In dev, you'll have the css-modules style loaded at the end, which is not good for overriding them by class names. I'll make the change for the docs because I need it for the examples. I did the same with the storybook to stay consistent.